### ISSUE TITLE
No way to open preview pane on Ubuntu 16.04 LTS

### ISSUE THREAD

**Issue by ramalho** [asciidoctor/atom-asciidoc-preview#261]
## Description

There's no documented way to open the asciidoc preview pane in Atom 1.24.1 on Ubuntu 16.4 LTS. because the shortcut <kbd>ctrl-shift-a</kbd> is bound to **File->Add project folder** and the shortcut <kbd>cmd-shift-a</kbd> simply writes an "A".

While this issue is not fixed, I'd appreciate workaround suggestions.

**Comment by ldez** [comment]
Thanks for your interest.

I use Ubuntu 16.04 LTS and the Atom 1.24.0 and the shortcut <kbd>ctrl+shift+a</kbd> works well.

I think that you have a conflict with the shortcuts of another package.

**Comment by ramalho** [comment]
Thanks for your reply, @ldez. However, I made a clean install of Atom. Only core packages an the asciidoc-assistant are installed.

**Comment by ldez** [comment]
To help you, you can:
- use the command palette <kbd>ctrl+p</kbd> and find "Keybing resolver: Toggle" to see the real key binding when you press <kbd>ctrl+shift+a</kbd>
- check the asciidoc-preview configuration:
  - <kbd>ctrl+,</kbd> to see the settings
  - in the section "Packages" find `asciidoc-preview`
  - in the "keybindings" section, verify if the checkbox `enable` is check.

**Comment by ramalho** [comment]
After restarting Atom several times I noticed it kept telling me that `asciidoc-assistant` as installing the other packages. That message persisted for hours, appearing every time I restarted Atom. So I manually installed the other packages one by one, and the problem went away. So the problem was not conflicting key bindings, but failed installation of the packages `asciidoc-assistant` should have installed.

The **File** menu still shows <key>ctrl-shift-a</key> as bound to **Add project folder**, but in fact that key combination opens the AsciiDoc preview pane.

Thanks for your help, @ldez!
